post_office/models.py

```diff
@@ -183,6 +183,9 @@ def dispatch(self, log_level=None,
             if not commit:
                 raise
 
+        if disconnect_after_delivery:
+            self.email_message().get_connection().close()
```

Comment:
```suggestion
            connections.close()
```
would also work, without failed tests, though it is not the exactly same behaviour as before. What is the preferred approach? Close all connection, rework the failed test or add an way to the ConnectionHandler to access cached connections without creating a new connection?